staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go

```diff
@@ -202,37 +202,7 @@ func dbHandler(ctx context.Context, twin *common.Twin, client *driver.Customized
 	switch twin.Property.PushMethod.DBMethod.DBMethodName {
 	// TODO add more database
 	case "influx":
-		dbConfig, err := dbInflux.NewDataBaseClient(twin.Property.PushMethod.DBMethod.DBConfig.Influxdb2ClientConfig, twin.Property.PushMethod.DBMethod.DBConfig.Influxdb2DataConfig)
-		if err != nil {
-			klog.Errorf("new database client error: %v", err)
-			return
-		}
-		dbClient := dbConfig.InitDbClient()
-		if err != nil {
-			klog.Errorf("init database client err: %v", err)
-			return
-		}
-		reportCycle := time.Duration(twin.Property.ReportCycle)
-		if reportCycle == 0 {
-			reportCycle = common.DefaultReportCycle
-		}
-		ticker := time.NewTicker(reportCycle)
-		go func() {
-			for {
-				select {
-				case <-ticker.C:
-					deviceData, err := client.GetDeviceData(visitorConfig)
-					if err != nil {
-						klog.Errorf("publish error: %v", err)
-						continue
-					}
-					sData, err := common.ConvertToString(deviceData)
-					if err != nil {
-						klog.Errorf("Failed to convert publish method data : %v", err)
-						continue
-					}
-					dataModel.SetValue(sData)
-					dataModel.SetTimeStamp()
+		dbInflux.DataHandler(ctx, twin, client, visitorConfig, dataModel)
```

Comment:
Do we need to add dbredis and dbtdengine too?

Reply:
yes, I pushed the new code.